yellowbrick/features/pcoords.py

```diff
@@ -231,3 +294,82 @@ def finalize(self, **kwargs):
         # Set the legend and the grid
         self.ax.legend(loc='best')
         self.ax.grid()
+
+
+##########################################################################
+## Normalized Static Parallel Coordinates Visualizer
+##########################################################################
+
+class NormalizedParallelCoordinates(ParallelCoordinates):
+    """
+    Parallel coordinates displays each feature as a vertical axis spaced
+    evenly along the horizontal, and each instance as a line drawn between
+    each individual axis.
+
+    Parameters
+    ----------
+
+    ax : matplotlib Axes, default: None
+        The axis to plot the figure on. If None is passed in the current axes
+        will be used (or generated if required).
+
+    features : list, default: None
+        a list of feature names to use
+        If a DataFrame is passed to fit and features is None, feature
+        names are selected as the columns of the DataFrame.
+
+    classes : list, default: None
+        a list of class names for the legend
+        If classes is None and a y value is passed to fit then the classes
+        are selected from the target vector.
+
+    color : list or tuple, default: None
+        optional list or tuple of colors to colorize lines
+        Use either color to colorize the lines on a per class basis or
+        colormap to color them on a continuous scale.
+
+    colormap : string or cmap, default: None
+        optional string or matplotlib cmap to colorize lines
+        Use either color to colorize the lines on a per class basis or
+        colormap to color them on a continuous scale.
+
+    vlines : boolean, default: True
+        flag to determine vertical line display
+
+    vlines_kwds : dict, default: None
+        options to style or display the vertical lines, default: None
+
+    kwargs : dict
+        Keyword arguments that are passed to the base class and may influence
+        the visualization as defined in other Visualizers.
+
+    Examples
+    --------
+
+    >>> visualizer = ParallelCoordinates()
+    >>> visualizer.fit(X, y)
+    >>> visualizer.transform(X)
+    >>> visualizer.poof()
+
+    Notes
+    -----
+
+    These parameters can be influenced later on in the visualization
+    process, but can and should be set as early as possible.
+    """
+
+    def draw(self, X, y, **kwargs):
+        """
+        Called from the fit method, this method creates the parallel
+        coordinates canvas and draws each instance and vertical lines on it.
+        """
+        # Convert from dataframe
+        if is_dataframe(X):
+            X = X.as_matrix()
+
+        # Normalize
+        maxes = X.max(axis=0)
+        mins = X.min(axis=0)
+        X_normed = (X - mins) / (maxes - mins)
```

Comment:
This is certainly a decent method of normalization, but there are quite a few methods and we may want to provide the user the option to choose normalization or standardization. I'd also prefer if we used the built in Sklearn Transformers, in this case: [MinMaxScaler](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html#sklearn.preprocessing.MinMaxScaler), [MinAbsScalar](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MaxAbsScaler.html#sklearn.preprocessing.MaxAbsScaler), the [Normalizer](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.Normalizer.html#sklearn.preprocessing.Normalizer), or the [StandardScalar](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html#sklearn.preprocessing.StandardScaler). This would work something like an internal pipeline, which other methods like the TSNE method or the PCA decomposition have done.